.github/workflows/ci.yml

```diff
@@ -106,8 +100,29 @@ jobs:
         command: build
         args: --release
 
-    - name: Test
-      if: matrix.libgccjit_version != 'libgccjit12.so'
+    - name: Test std_tests
+      run: |
+        # Enable backtraces for easier debugging
+        export RUST_BACKTRACE=1
```

Comment:
Perhaps we should enable this at the top level to avoid having to specify it multiple times?